docs/semgrep-code/notifications.md

```diff
@@ -402,40 +403,146 @@ If you are using GitHub Actions to run Semgrep, no extra changes are needed to g
 
 ### GitLab merge request comments
 
+:::info Prerequisites
+- GitLab MR comments are only available to logged-in Semgrep Cloud Platform users.
```

Comment:
```suggestion
- GitLab MR comments are only available to Semgrep Cloud Platform users.
```

After you set up PR/MR comments people do not need to be logged in SCP as they will see it in their SCM.